app-arch/lzfse/lzfse-1.0.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+CMAKE_INSTALL_PREFIX=""
```

Comment:
Why are we setting this? That's quite unusual.

Reply:
I don't recall why these were necessary. Could have been to work around something in `cmake.eclass`, but they're not necessary anymore.